app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -50,6 +50,15 @@ def mfa_enroll_totp(success:)
       )
     end
 
+    # Tracks when the user has attempted to enroll the WebAuthn-Platform MFA method to their account
+    # @param [Boolean] success
+    def mfa_enroll_webauthn_biometric(success:)
```

Comment:
Based on [this Yubico documentation](https://developers.yubico.com/WebAuthn/WebAuthn_Developer_Guide/Platform_vs_Cross-Platform.html)

> Note that a biometric sensor is not *required* for a platform authenticator. The point is that the authenticator must support the type of input required, which could just as well be PIN for user verification, or indeed, in the case where there is no user verification at all, the TPM would be used as a U2F style second factor authenticator.

I think we can be more true to what we accept if we leave this event as `webauthn_platform`, and we can document that Face/Touch ID are the most common implementations today

Reply:
Cool, will do - thanks for reviewing that for me!